### ISSUE TITLE
AttributeError: type object 'datasketches.theta_sketch' has no attribute 'deserialize'

### ISSUE THREAD

**Issue by CarreauClement** [apache/datasketches-cpp#306]
I'm using the following version in Python
Name: datasketches
Version: 3.5.0

Trying to follow this tutorial : https://github.com/apache/datasketches-cpp/blob/master/python/jupyter/ThetaSketchNotebook.ipynb

But the method theta_sketch.deserialize does not exist.
How can I deserialize bytes from a serialized theta sketch ?

Thanks

**Comment by CarreauClement** [comment]
I found it, the file is just no up to date with the latest version of the lib.
We should use the following 

```
from datasketches import compact_theta_sketch
new_sk1 = compact_theta_sketch.deserialize(sk1_bytes)
```

**Comment by jmalkin** [comment]
Oops, you found the solution and closed this while I was confirming locally that we'd moved it to the compact form only.

Yes, the tutorial is clearly outdated now. Thanks for catching that!

**Comment by jmalkin** [comment]
Fixed in #307
